packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-request-sender.spec.ts

```diff
@@ -39,6 +39,16 @@ describe('PaypalCommerceRequestSender', () => {
             }));
         });
 
+        it('deleteCart', async () => {
```

Comment:
Can you ad a test for `setShippingOptions`?

Reply:
added